lib/web_ui/dev/test_runner.dart

```diff
@@ -60,6 +60,10 @@ class TestCommand extends Command<bool> {
 
     _copyTestFontsIntoWebUi();
     await _buildHostPage();
+    if(io.Platform.isWindows) {
```

Comment:
nit: add space after `if` (or just run `dartfmt` on the whole thing)

Reply:
done.